sdk/servicebus/service-bus/test/public/atomManagement.spec.ts

```diff
@@ -1724,7 +1718,7 @@ describe(`createRule() using different variations to the input parameter "ruleOp
   // Rule tests
   const createRuleTests: {
     testCaseTitle: string;
-    input: Omit<CreateRuleOptions, "name"> | undefined;
+    input: Omit<Required<CreateSubscriptionOptions>["defaultRuleOptions"], "name"> | undefined;
```

Comment:
The reason for this change is.. `CreateRuleOptions` is not public?

Reply:
Yes because it's not public